shell/common/rasterizer.cc

```diff
@@ -605,6 +605,7 @@ RasterStatus Rasterizer::DrawToSurfaceUnsafe(
     if (damage) {
       submit_info.frame_damage = damage->GetFrameDamage();
       submit_info.buffer_damage = damage->GetBufferDamage();
+      submit_info.clip_rect = damage->GetClipRect();
```

Comment:
`buffer_damage` contains the area of buffer that has been written to, which should be identical with the clip rect. Does this need to be passed separately?

Reply:
I wonder if I got this mixed up due to a separate math problem and assumed they were different. Let me double check that.

Reply:
Yup, got things mixed. Removed clip rect in favor of buffer damage. Thanks!